### PR TITLE
Setup demo acs

### PR DESCRIPTION
**Pull request by MarcAntoine-Arnaud**

### CODE REVIEW

File: assets/src/app/models/s3.ts [hunk 1]
```diff
@@ -4,5 +4,6 @@ export class S3Configuration {
   access_key: string
   region: string
   bucket: string
+  vod_endpoint?: string
```
Comment (valnoel) [inline assets/src/app/models/s3.ts]: Could you add some documentation to explain what is this field about? 🙏

File: assets/src/app/order/orders.component.less [hunk 2]
```diff
@@ -10,6 +10,43 @@
...
+.link.disabled {
+  color: @grey;
+  opacity: 0.8;
+}
+
+.percentage {
+  margin: auto;
+  text-align: center;
+  font-size: 0.7em;
+}
```
Comment (valnoel) [inline assets/src/app/order/orders.component.less]: ↩️

File: assets/src/app/services/workflow.service.ts [hunk 3]
```diff
@@ -5,12 +5,13 @@ import { Observable, of } from 'rxjs'
...
 import {WorkflowPage, WorkflowData, WorkflowHistory} from '../models/page/workflow_page'
-import {Flow, Step, Workflow, WorkflowEvent} from '../models/workflow'
+import {Step, Workflow, WorkflowEvent} from '../models/workflow'
 
 @Injectable()
 export class WorkflowService {
   private workflowUrl = '/api/workflow'
-  private workflowsUrl = '/api/workflows'
+  private workflowsUrl = '/api/step_flow/workflows'
+  private statisticsUrl = '/api/step_flow/workflows_statistics'
```
Comment (valnoel) [inline assets/src/app/services/workflow.service.ts]: These fields could be `const`

File: config/dev.exs [hunk 4]
```diff
@@ -51,8 +51,9 @@ config :ex_backend, ExBackendWeb.Endpoint,
 
 # Do not include metadata nor timestamps in development logs
 config :logger, :console, format: "[$level] $message\n"
-# config :logger, level: :debug
-config :logger, level: :info
+config :logger, level: :debug
+# config :logger, level: :info
+# config :logger, level: :warn
```
Comment (valnoel) [inline config/dev.exs]: Is this really useful keeping this comments? :P

File: config/dev.exs [hunk 5]
```diff
@@ -102,19 +111,22 @@ config :ex_backend,
...
-  docker_container_amqp_username: "guest",
-  docker_container_amqp_password: "guest"
+  docker_container_amqp_username: "mediacloudai",
+  docker_container_amqp_password: "mediacloudai"
 
 config :amqp,
-  hostname: "127.0.0.1",
-  username: "guest",
-  password: "guest"
+  hostname: "192.168.99.101",
```
Comment (valnoel) [inline config/dev.exs]: What is this IP corresponding to?

File: lib/ex_backend/workflow/definition/francetv_acs.ex [hunk 6]
```diff
@@ -19,37 +21,138 @@ defmodule ExBackend.Workflow.Definition.FrancetvAcs do
...
         parameters: [
           %{
             id: "input_filter",
             type: "filter",
             default: %{ends_with: [".mp4", ".isma"]},
             value: %{ends_with: [".mp4", ".isma"]}
+          },
+          %{
+            id: "command_template",
+            type: "string",
```
Comment (valnoel) [inline lib/ex_backend/workflow/definition/francetv_acs.ex]: Shouldn't it be a `template` instead?

File: lib/ex_backend/workflow/definition/francetv_acs.ex [hunk 7]
```diff
@@ -19,37 +21,138 @@ defmodule ExBackend.Workflow.Definition.FrancetvAcs do
...
         parameters: [
           %{
             id: "input_filter",
             type: "filter",
             default: %{ends_with: [".ttml", ".wav"]},
             value: %{ends_with: [".ttml", ".wav"]}
           },
+          %{
+            id: "command_template",
+            type: "string",
```
Comment (valnoel) [inline lib/ex_backend/workflow/definition/francetv_acs.ex]: Shouldn't it be a `template` instead?

File: lib/ex_backend/workflow/definition/francetv_acs.ex [hunk 8]
```diff
@@ -19,37 +21,138 @@ defmodule ExBackend.Workflow.Definition.FrancetvAcs do
...
+          },
+          %{
+            id: "audio_path",
+            type: "select_input",
+            default: %{ends_with: [".wav"]},
+            value: %{ends_with: [".wav"]}
+          },
+          %{
+            id: "destination_filename",
+            type: "template",
```
Comment (valnoel) [inline lib/ex_backend/workflow/definition/francetv_acs.ex]: Shouldn't it be a `string` instead?

File: lib/ex_backend/workflow/definition/francetv_acs.ex [hunk 9]
```diff
@@ -61,147 +164,79 @@ defmodule ExBackend.Workflow.Definition.FrancetvAcs do
...
+        parameters: [
+          %{
+            id: "action",
+            type: "string",
+            default: "remove",
+            value: "remove"
+          },
+          %{
+            id: "source_paths",
+            type: "array_of_templates",
```
Comment (valnoel) [inline lib/ex_backend/workflow/definition/francetv_acs.ex]: ⭐️  🚀 ⭐️

File: lib/ex_backend/workflow/workflows.ex [hunk 10]
```diff
@@ -39,6 +39,8 @@ defmodule ExBackend.Workflows do
 
     offset = page * size
 
+    IO.inspect(params)
```
Comment (valnoel) [inline lib/ex_backend/workflow/workflows.ex]: Is this really useful? 😉

File: lib/ex_backend/workflow/definition/francetv_acs.ex (hunk 6 above)
Comment (MarcAntoine-Arnaud) [reply lib/ex_backend/workflow/definition/francetv_acs.ex]: nope, it's the parameter passed to the worker.
Type template is not send to workers, it's just used to generate variable from a template in step_flow.

File: lib/ex_backend/workflow/workflows.ex (hunk 10 above)
Comment (MarcAntoine-Arnaud) [reply lib/ex_backend/workflow/workflows.ex]: sometimes :-P

**Pull request merged**